src/ng/directive/ngOptions.js

```diff
@@ -591,7 +591,7 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
           element.label = option.label;
           element.textContent = option.label;
         }
-        if (option.value !== element.value) element.value = option.selectValue;
+        element.value = option.selectValue;
```

Comment:
Interestingly, `option.value` will always be undefined, so this was a redundant.
I wonder what the intenton was, though.